base/docs/Docs.jl

```diff
@@ -201,7 +201,7 @@ end
 
 doc(f, ::Method) = doc(f)
 
-# Modules
+# Modules
```

Comment:
I assume that must be my fault, although I have no idea why an editor would insert one of those.